migrations/202210131714933_add_schema_comments.up.sql

```diff
@@ -0,0 +1,281 @@
+COMMENT ON COLUMN public.git_refs.name IS 'name of the ref';
+COMMENT ON COLUMN public.git_refs.remote IS 'remote of the ref';
+COMMENT ON COLUMN public.git_refs.target IS 'target of the ref';
+COMMENT ON COLUMN public.git_refs.type IS 'type of the ref';
+COMMENT ON COLUMN public.git_refs.tag_commit_hash IS 'hash of the commit for refs that or of type tag';
+COMMENT ON COLUMN public.git_refs._mergestat_synced_at IS 'timestamp when record was synced into the MergeStat database';
+
+COMMENT ON view public.git_branches IS 'view of git refs of type branch of a repo';
+COMMENT ON COLUMN public.git_branches.repo_id IS 'foreign key for public.repos.id';
+COMMENT ON COLUMN public.git_branches.hash IS 'hash of the commit for refs that are not of type tag';
+COMMENT ON COLUMN public.git_branches.name IS 'name of the ref';
+COMMENT ON COLUMN public.git_branches.remote IS 'remote of the ref';
+COMMENT ON COLUMN public.git_branches.target IS 'target of the ref';
+COMMENT ON COLUMN public.git_branches.type IS 'type of the ref';
+COMMENT ON COLUMN public.git_branches.tag_commit_hash IS 'hash of the commit for refs that or of type tag';
+COMMENT ON COLUMN public.git_branches._mergestat_synced_at IS 'timestamp when record was synced into the MergeStat database';
+
+COMMENT ON view public.git_tags IS 'view of git refs of type tag of a repo';
+COMMENT ON COLUMN public.git_tags.repo_id IS 'foreign key for public.repos.id';
+COMMENT ON COLUMN public.git_tags.hash IS 'hash of the commit for refs that are not of type tag';
+COMMENT ON COLUMN public.git_tags.name IS 'name of the ref';
+COMMENT ON COLUMN public.git_tags.remote IS 'remote of the ref';
+COMMENT ON COLUMN public.git_tags.target IS 'target of the ref';
+COMMENT ON COLUMN public.git_tags.type IS 'type of the ref';
+COMMENT ON COLUMN public.git_tags.tag_commit_hash IS 'hash of the commit for refs that or of type tag';
+COMMENT ON COLUMN public.git_tags._mergestat_synced_at IS 'timestamp when record was synced into the MergeStat database';
+
+COMMENT ON TABLE public.github_issues IS 'issues of a GitHub repo';
+COMMENT ON COLUMN public.github_issues.repo_id IS 'foreign key for public.repos.id';
+COMMENT ON COLUMN public.github_issues.author_login IS 'login of the author of the issue';
+COMMENT ON COLUMN public.github_issues.body IS 'body of the issue';
+COMMENT ON COLUMN public.github_issues.closed IS 'boolean to determine if the issue is closed';
+COMMENT ON COLUMN public.github_issues.closed_at IS 'timestamp of when the issue was closed';
+COMMENT ON COLUMN public.github_issues.created_via_email IS 'boolean to determine if the issue was created via email';
+COMMENT ON COLUMN public.github_issues.database_id IS 'GitHub database_id of the issue';
+COMMENT ON COLUMN public.github_issues.editor_login IS 'login of the editor of the issue';
+COMMENT ON COLUMN public.github_issues.includes_created_edit IS 'boolean to determine if the issue was edited and includes and edit with the creation data';
```

Comment:
```suggestion
COMMENT ON COLUMN public.github_issues.includes_created_edit IS 'boolean to determine if the issue was edited and includes and edit with the creation data';
```

not sure of a good phrasing exactly - unclear a bit on what the column actually is too 😄, I will look it up in the GitHub API too

Reply:
This is the wording in the [GitHub docs](https://docs.github.com/en/graphql/reference/objects)
`Check if this comment was edited and includes an edit with the creation data.`